src/Ledger/Conway/Specification/PoolReap/Properties/Computational.agda

```diff
@@ -0,0 +1,37 @@
+{-# OPTIONS --safe #-}
+
+open import Ledger.Prelude
```

Comment:
For consistency, should you move `open import Ledger.Prelude` inside the module?